### PR TITLE
Hotfix 13/modal updates

### PR DESCRIPTION
**Pull request by johnrobertmcc**
Removed last line
updated class names
Refactored to be called from App, deleted Footer file

_Unsure how to add a close button without resetting the "completed" files. it opens based on the state_

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
+  "parserOptions": {
+    "ecmaVersion": 2019,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
     }
-}
\ No newline at end of file
+  },
+  "extends": ["plugin:react/recommended", "prettier"],
```
Comment (michealengland) [inline .eslintrc]: After testing this locally with the original rules `"eslint:recommended", "plugin:react/recommended", "prettier"`, I believe we should keep this as it originally was. Meanwhile, the `no-debugger` rule makes sense to move forward on as it's either on or off. There is no `warning` option for this feature that I could locate.

**Review by michealengland — CHANGES_REQUESTED** [review]
Other than reverting a minor change, I'm am good with these changes as is.

**Comment by johnrobertmcc** [comment]
@michealengland eslint recommended readded

**Review by michealengland — APPROVED** [review]
👍

**Pull request merged**